CHANGES.txt

```diff
@@ -5,6 +5,13 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Updated crate-admin to ``1.1.0`` which includes the following change:
+
+    - Breaking: Updated the admin-ui build to be compatible with future crate
+      versions which will serve the admin-ui from root `/`.
+
+ - Serve admin ui from `/`
```

Comment:
Guess the user-facing breaking change is that the admin-ui can no longer be accessed via `/_plugin/crate-admin/#/`  but only `/` or `/admin`